web/src/engine/websites/MangaHubRU.ts

```diff
@@ -30,11 +30,11 @@ export default class extends DecoratableMangaScraper {
     }
 
     public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
-        const url = new URL(manga.Identifier.replace('/title/', '/chapters/'), this.URI);
+        const url = new URL(manga.Identifier + '/chapters', this.URI);
```

Comment:
We now can compose an url to gather chapters using "Common.Chapter" decorators . Wwhile correct this syntax is deprecated. Thank for for taking the time to submit a PR, a fix is coming